mappings/paulscode/sound/SoundSystem.mapping

```diff
@@ -0,0 +1 @@
+CLASS paulscode/sound/SoundSystem paulscode/sound/SoundSystem
```

Comment:
Why does this have a mapping? It's not obfuscated.

Reply:
thing I ctrl oed it